share/tunnel/tunnel_in_proxy_udp.go

```diff
@@ -100,15 +100,19 @@ func (u *udpListener) runInbound(ctx context.Context) error {
 			if strings.HasSuffix(err.Error(), "EOF") {
 				continue
 			}
-			return u.Errorf("inbound-udpchan: %w", err)
+			u.Errorf("inbound-udpchan: %w", err)
+			// wait for a short while for the ssh connection
+			time.Sleep(time.Duration(100) * time.Millisecond)
```

Comment:
instead of sleeping, can use channels to know when the ssh connection is ready?

Reply:
The reason for adding sleep before is that, when the connection is lost, getSSH() will return nil immediately, and the runOutbound() loop will take up a lot of memory and cause the process to be killed.

Reply:
Ohh the loop spins? I haven’t seen that happen before though that sounds like a bug, where the real fix might be to  block with a waitgroup

Reply:
After connection lost, `activatingConnWait()` cannot block, because `t.activatingConn.DoneAll()` is called.
Removing `DoneAll()` fixes the issue.